packages/hathor-rpc-handler/src/rpcMethods/getAddress.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { HathorWallet } from '@hathor/wallet-lib';
+import { AddressRequestClientResponse, ConfirmationPromptTypes, GetAddressRpcRequest, PromptHandler } from '../types';
+import { NotImplementedError, PromptRejectedError } from '../errors';
+import { validateNetwork } from '../helpers';
+
+/**
+ * Gets an address based on the provided rpcRequest and wallet.
+ *
+ * @param rpcRequest - The RPC request containing the parameters for getting an address.
+ * @param wallet - The wallet instance to use for retrieving the address.
+ * @param promptHandler - A function to handle prompts for user confirmation.
+ *
+ * @returns The address retrieved based on the request parameters.
+ *
+ * @throws {NotImplementedError} - If the request type is 'full_path', which is not implemented.
+ * @throws {PromptRejectedError} - If the user rejects the address confirmation prompt.
+ */
+export async function getAddress(
+  rpcRequest: GetAddressRpcRequest,
+  wallet: HathorWallet,
+  promptHandler: PromptHandler,
+) {
+  validateNetwork(wallet, rpcRequest.params.network);
+
+  const { type, index } = rpcRequest.params;
+  let address: string;
+
+  switch (type) {
+    case 'first_empty':
+      address = await wallet.getCurrentAddress();
+    break;
+    case 'full_path':
+      throw new NotImplementedError();
```

Comment:
When will this be implemented? It's kind of easy to get it from the wallet object and some utils methods.